net2/HostManager.js

```diff
@@ -1008,6 +1008,9 @@ module.exports = class HostManager {
         if(firstBinding) {
           json.firstBinding = firstBinding
         }
+        if (json.mode === "spoof" || json.mode === "none") {
+          json.routerChangeFlag = await rclient.getAsync("sys:router:change");
```

Comment:
Client should be able to know what interfaces have problems so it can proceed on compatibility test.